runtime/codert_vm/jswalk.c

```diff
@@ -1793,7 +1793,8 @@ jitGetOwnedObjectMonitors(J9StackWalkState *walkState)
 			if (liveMonitorMap) {
 				U_8 *inlineMonitorMask;
 
-				if (inlineMonitorMask = getMonitorMask(gcStackAtlas, inlinedCallSite)) {
+				inlineMonitorMask = getMonitorMask(gcStackAtlas, inlinedCallSite);
```

Comment:
Why not move this assignment into the initializer of `inlineMonitorMask`? (likewise for similar situations below)

Reply:
Good catch.  I missed that when fixing the compile error.